### ISSUE TITLE
'Releasing' the cursor

### ISSUE THREAD

**Issue by olneyMatt** [amcharts/amcharts5#237]
Hi,

When the user drops a pin in a `MapChart`, I'm setting the `positionX` and `alwaysShow` setting of an`XYCursor` in an associated `XYChart`.

When they move out of the `MapChart`, I want the `XYCursor` of the `XYChart` to revert to "normal" behaviour (i.e. follow the mouse, show tooltips, hover etc.)

I can trigger the correct event, but I can't figure out how to change the cursor settings to achieve this.

I've tried setting `positionX` and `alwaysShow` to `undefined`, but the cursor stays stuck where I put it.

Can you offer any suggestions please?

Thanks
Matt

**Comment by martynasma** [comment]
Technically, this should work, but doesn't:

```JavaScript
cursor.setAll({
  positionX: undefined,
  alwaysShow: false
});
```

We'll fix it in the next version and let you know here.

**Comment by olneyMatt** [comment]
Great - thanks so much for your help. 👍

**Comment by martynasma** [comment]
Fixed in [5.1.1](https://github.com/amcharts/amcharts5/releases/tag/5.1.1).

## [5.1.1] - 2022-01-31

## Added
- New method `remove(key)` on Template's adapaters.
- Each chart-type package now exports its default theme as `DefaultTheme`. E.g. `am5xy.DefaultTheme`, `am5flow.DefaultTheme`, etc.
- TypeScript: Settings interfaces for all objects are now being exposed via bundled re-export files like `index.ts`, `xy.ts`, etc.
- New class `am5.Modal`.
- New `Container` setting: `reverseChildren`. If set to `true`, will start rendering from the last child, effectively flipping the order in how children of the cotainer are laid out. Useful for [right-to-left setups](
https://www.amcharts.com/docs/v5/concepts/locales/right-to-left-support/).

## Changed
- Charts on iOS devices will now automatically use lesser resolution to work around canvas memory restriction introduced in iOS 15 Safari.
- Charts will now use less built-in layers, and thus less `<canvas>` elements by default, reducing memory consumption.
- All low/high-related settings and fields moved from `CandlestickSeries` to `XYSeries`. Sometimes it could be useful to have low/high on other series even if they are not shown visually.

## Fixed
- `XYCursor` was not being hidden when its `positionX` or `positionY` was changed to `undefined` and `alwaysShow` was set to `false`.
- Auto-wrapped right-aligned labels were being rendered in wrong position.
- Auto-wrapped labels were not being trimmed for whitespace on each line.
- `dropFromPrevious` and `riseFromPrevious` states on `ColumnSeries` were not working properly on a column after a gap in data in case previous data item was outside of selection.
- Emoji characters were not being displayed on circular labels.

[Full change log](https://github.com/amcharts/amcharts5/blob/master/packages/shared/CHANGELOG.md).

[Download options](https://www.amcharts.com/download/).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
